Fix: typos and errors (#9)
moT01/.learn-git-by-building-an-sql-reference-object: TUTORIAL.md

### 1570.1

> You can see that the latest commit is at the bottom here. Be careful not to change the wrong commits. One of the options is `r, reword = use commit, but edit the commit message`. Replace `pick` with an `r` next to the commit with the message `feat: add column reference` to reword the message, it's the very first commit you added to this branch. When you are done, save the file and exit Nano. Git will put you in an another `nano` instance to `reword` commit message for the commit you just chose. Do not change anything in it just yet.

Reviewer: ```suggestion
You can see that the latest commit is at the bottom here. Be careful not to change the wrong commits. One of the options is `r, reword = use commit, but edit the commit message`. Replace `pick` with an `r` next to the commit with the message `feat: add column reference` to reword the message, it's the very first commit you added to this branch. When you are done, save the file and exit Nano. Git will put you in another Nano instance to reword the commit message. Don't change anything in it yet.
```

### 1572.1

> You are already in an `nano` instance, editing the commit meassage for the selected commit. Add an `s` at the end of the commit message so it is `feat: add column references`. When you are done, save the file and exit Nano.

Reviewer: ```suggestion
Git is waiting for you to submit the commit message. Add an `s` at the end of the commit message so it is `feat: add column references`. When you are done, save the file and exit Nano.
```

### 1590.1

> Squashing commits means that you will take a bunch of commits and turn them into one. This is helpful to keep your commit history clean and something you want try to do. Replace `pick` with an `s` next to all your commits except the one with the message `feat: add column references`. When you are done, save and exit the file. You will find yourself in another instance of `nano`. Don't change aything just yet.

Reviewer: ```suggestion
Squashing commits means that you will take a bunch of commits and turn them into one. This is helpful to keep your commit history clean and something you want try to do. Replace `pick` with an `s` next to all your commits except the one with the message `feat: add column references`. When you are done, save and exit the file. You will find yourself in another instance of Nano. Don't change anything in it yet.
```
